integration_tests/run_pyspark_from_build.sh

```diff
@@ -29,11 +29,13 @@ else
     if [ -d "$LOCAL_JAR_PATH" ]; then
         CUDF_JARS=$(echo "$LOCAL_JAR_PATH"/cudf-*.jar)
         PLUGIN_JARS=$(echo "$LOCAL_JAR_PATH"/rapids-4-spark_*.jar)
+        # TODO - this should really only pick up the specific version being tested
```

Comment:
Is there a follow on issue for this?

Reply:
its subitem in https://github.com/NVIDIA/spark-rapids/issues/3223